src/kmergraph.cpp

```diff
@@ -187,33 +164,35 @@ void KmerGraph::remove_shortcut_edges() {
     for (const auto &n : nodes) {
         //cout << n.first << endl;
         for (const auto &out : n->outNodes) {
-            for (auto next_out = out->outNodes.begin(); next_out != out->outNodes.end();) {
+            auto outNodeAsSharedPtr = out.lock();
```

Comment:
I know it sounds *very* pedantic but could we please stick to the same convention for variable/function/class names - i.e snake_case rather than snakeCase. This should be easy enough to fix in CLion.

Reply:
Done in https://github.com/leoisl/pandora/commit/eb6e068598fc1bb420e63f63819c27ed8bf9e600 and https://github.com/leoisl/pandora/commit/f15721908f426e0f1d5840f1ad2097420b648c7b